src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -198,12 +198,14 @@ public void fillInnerParts() {
         mePanel = new MePanel(FXCollections.observableArrayList(security.getUser()));
         mePanelPlaceholder.getChildren().add(mePanel.getRoot());
 
-        personListPanel = new PersonListPanel(logic.getOtherList(security.getUser()));
-        personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
-
-        friendListPanel = new FriendListPanel(logic.getFriendList(security.getUser()));
```

Comment:
You're not using logic.getFriendList() anymore? Or does getFilteredPersonList() does the same too?

Reply:
If you were to refer to #56 the cascade of system tests is triggered with the removal of `getFilteredPersonList()` I'm still working on a way to fix it for now

Reply:
Also refer to the description of #68